frontend/src/components/buttons/GroupButtons.vue

```diff
@@ -74,6 +74,17 @@ const leaveGroupAndRedirect = async () => {
     await leaveGroup({ groupId: group.value.id });
     router.push(`/project/${project.value.id}/groups`);
 };
+
+const removeGroupAndRedirect = async () => {
+    await removeGroup({ groupId: group.value.id });
+    router.push(`/project/${project.value.id}/groups`);
```

Comment:
waarom wordt hier een redirect gedaan?

Reply:
Omdat diezelfde knop voor de teacher ook beschikbaar is in de groepspagina en anders blijf je ongemakkelijk op die pagina bij na verwijderen. Geeft ook geen problemen wanneer je een groep verwijdert vanuit de groups pagina dus zag er geen probleem mee.